src/pages/profitSharing/index.js

```diff
@@ -0,0 +1,133 @@
+import React, { useEffect } from "react";
```

Comment:
We have to refactor this component to reuse it in the other places and avoid repeating the code. We can create a separate task if this needs to be merged first or if we need to discuss how to approach that.

Reply:
I left this myself, I have some other components to refactor as well so I will do them slowly slowly on daily basis

Reply:
I have done small refactoriing for now. and I think that enough for this now.

Reply:
Ok then feel free to merge as it is for now.